core/src/main/scala/replpp/package.scala

```diff
@@ -132,9 +109,6 @@ package object replpp {
   private def lines(str: String): Seq[String] =
     str.split(lineSeparator)
 
```

Comment:
```suggestion
```

Seems to be dead code now. (I was looking for uses of this, because splitting by a system-specific line separator might break for files with non-native line endings.)

Reply:
indeedly, thanks